### PR TITLE
[Beta] Add tax components

### PR DESCRIPTION
**Pull request by kevinpeters-stripe**
Add `tax-registrations` and `tax-settings` components.

### CODE REVIEW

File: src/shared.ts [hunk 1]
```diff
@@ -42,7 +44,9 @@ export const componentNameMapping: Record<
   "financial-account": "stripe-connect-financial-account",
   "financial-account-transactions":
     "stripe-connect-financial-account-transactions",
-  documents: "stripe-connect-documents"
+  documents: "stripe-connect-documents",
+  "tax-registrations": "stripe-connect-tax-registrations",
```
Comment (jorgea-stripe) [inline src/shared.ts:48]: To confirm, there aren't any props on these components yes?

**Review by jorgea-stripe — APPROVED** [review]

File: src/shared.ts (hunk 1 above)
Comment (kevinpeters-stripe) [reply src/shared.ts:48]: Yeah! No props!

**Comment by kevinpeters-stripe** [comment]
Will merge the PR in react-connect-js tomorrow: https://github.com/stripe/react-connect-js/pull/83

**Review by jorgea-stripe — APPROVED** [review]
LGTM

**Review by jorgea-stripe — APPROVED** [review]

**Pull request merged**